### PR TITLE
rm: various tweaks for clarity; add -v

### PR DESCRIPTION
**Pull request by waldyrious**
changed some descriptions and tokens to make the meaning of each command clearer, and facilitate memorization of the flags
added the -v (verbose) option

### CODE REVIEW

File: pages/common/rm.md [hunk 1]
```diff
@@ -4,16 +4,20 @@
...
-- Remove directory without prompt:
+- Forcibly remove a directory, without prompting for confirmation or showing error messages:
 
-`rm -rf {{/path/to/folder}}`
+`rm -rf {{path/to/folder}}`
 
-- Prompt before every removal:
+- Interactively remove multiple files, with a prompt before every removal:
 
 `rm -i {{\*}}`
```
Comment (waldyrious) [inline pages/common/rm.md]: No idea. I'd be in favor of using a more intuitive pattern. Any suggestions?

Comment (agnivade) [inline pages/common/rm.md]: I checked the page for unzip. It has something like this - `rm -i {{file(s)}}`.

Comment (waldyrious) [inline pages/common/rm.md]: Sounds good to me, I'll change accordingly.

**Comment by agnivade** [comment]
:+1:

**Pull request merged**